### PR TITLE
Marketing

### PR DESCRIPTION
**Pull request by nganhquoc95**
Marketing Screen 19 - 23.

### CODE REVIEW

File: app/assets/scss/_color_status.scss [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Color text for status
+\*---------------------------*/
+
+/**
+ * Color status Active
+ */
+.text-active {
+  color: #2fd475;
+}
+
```
Comment (chungphuoc) [inline app/assets/scss/_color_status.scss]: too many lines break.

File: app/assets/scss/_color_status.scss [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+
+/**
+ * Color status Pending
+ */
+.text-pending {
+  color: #f5a623;
+}
+
```
Comment (chungphuoc) [inline app/assets/scss/_color_status.scss]: too many lines break.

File: app/assets/scss/research/product-database.scss [hunk 3]
```diff
@@ -148,13 +166,17 @@ Common Product Database
...
+      padding-left: 0;
+    }
   }
 
 
   /**
    * Favorite product
    */
-  tbody > tr > .favorite {
+  & > tbody > tr > td.favorite {
```
Comment (chungphuoc) [inline app/assets/scss/research/product-database.scss]: should add class name for the tbody

File: app/assets/styles/marketing/ab_testing.css [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+    margin-right: -1px;
+    color: #212b36;
+    border-top-left-radius: 6px;
+    border-top-right-radius: 6px;
+    background-color: #ffffff;
+    border: 1px solid #ddd; }
+    .ab-testing .panel-heading__dialog .wrapper-selectpicker {
+      margin-top: 30px;
+      margin-bottom: 175px; }
+  .ab-testing .panel-body__dialog {
```
Comment (chungphuoc) [inline app/assets/styles/marketing/ab_testing.css]: should be like this 
.ab-testing .panel-body__dialog {
   padding: 40px; 
}
Please correct all the format style in this files or other files.

Comment (nganhquoc95) [reply app/assets/styles/marketing/ab_testing.css]: The files name has the extension ".CSS", Are files that have been rendered from the SCSS files. I will add that to the ".ingrore" file.

Comment (chungphuoc) [reply app/assets/styles/marketing/ab_testing.css]: ok, and you need to fix the conflict files.

Comment (nganhquoc95) [reply app/assets/styles/marketing/ab_testing.css]: Wow. There are the conflict files :(.

**Pull request merged**